code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/local/LocalAuthTokenCreator.java

```diff
@@ -68,4 +68,14 @@ public void reset() {
     public String getName() {
         return LocalAuthConstants.TOKEN_CREATOR;
     }
+
+    @Override
+    public Token getCurrentToken() {
+        return localAuthTokenUtils.retrieveCurrentToken();
+    }
+
+    @Override
+    public String getTokenJWTValue() {
+        return localAuthTokenUtils.getJWT();
+    }
```

Comment:
We may not need this if you start setting JWT to the token returned by getCurrentToken()